gatsby_fp/src/components/fields.js

```diff
@@ -0,0 +1,89 @@
+import React from "react"
+import GenInfoTable from "./fieldComponents/GenInfoTable"
+import Education from "./fieldComponents/Education"
+import ProfService from "./fieldComponents/ProfService"
+import UnivService from "./fieldComponents/UnivService"
+import ProfSocieties from "./fieldComponents/ProfSocieties"
+import IntellCont from "./fieldComponents/IntellCont"
+import GeneralSection from "./fieldComponents/GeneralSection"
+
+export default function Fields({ facultyData }) {
+  return (
+    <>
+      <h1 className="biotitle">
+        {facultyData.pf_first_name} {facultyData.pf_last_name}
+      </h1>
+      {/* still need to conditionally render fields for this component */}
```

Comment:
Comments like this should start with the string `@todo` so they can be more easily located. In this case, is it even relevant? I do see conditionals throughout GenInfoTable.js

Reply:
Removed, irrelevant tag